Improve contributing guidelines (#54)
aim-rsf/browseMetadata: README.md

> ## Contributing changes

Reviewer: ```suggestion
```

## Contributing

> Your contributions are what make the browseMetadata community amazing! Whether it's fixing bugs, adding new features, or improving documentation, we welcome your involvement.

Reviewer: ```suggestion
We warmly welcome contributions to the browseMetadata project. Whether it's fixing bugs, adding new features, or improving documentation, we welcome your involvement. 
```
